src/backend/TrafficCourts/Citizen.Service/Validators/Rules/CountSectionRule.cs

```diff
@@ -22,13 +22,17 @@ public CountSectionRule(Field field, Services.ILookupService lookupService) : ba
 
     public override void Run()
     {
-        string? value = Field.Value;
-        if (!String.IsNullOrEmpty(value))
+        if (!String.IsNullOrEmpty(Field.Value))
         {
-            value = Field.Value = Regex.Replace(value, @"\s+", "");
-            IEnumerable<Statute> statutes = _lookupService.GetStatutes(value.ToLower().Trim());
-            if (statutes is null || !statutes.Any()) {
-                AddValidationError(String.Format(ValidationMessages.CountSectionInvalid, Field.Value));
+            Field.Value = Regex.Replace(Field.Value, @"\s+", "");
+            Field.Value = Regex.Replace(Field.Value, @"^\$$", "");
+            if (!String.IsNullOrEmpty(Field.Value))
+            {
+                IEnumerable<Statute> statutes = _lookupService.GetStatutes(Field.Value.ToLower().Trim());

```

Comment:
Would it make sense to move this logic into the look up service?  like `bool StatusExists(string value)`, then the lookup service can deal with trim, etc. However, it is ok as it is.

Reply:
Good idea.  Yes, logic should be in the service, not validator.  Moved ToLower and Trim to service.